Library/Homebrew/cmd/command.rb

```diff
@@ -21,11 +21,7 @@ def command
   def internal_command_path(cmd)
```

Comment:
I may be wrong here, but this seem to do the same as `command_path` in help.rb now, but much nicer. If it does, we can just use this one instead, but I'm not sure where it should live.